src/Pact/Native.hs

```diff
@@ -144,7 +144,10 @@ enforceDef = defNative "enforce" enforce
   where
 
     enforce :: NativeFun e
-    enforce i as = runSysOnly $ gasUnreduced i as $ mapM reduce as >>= enforce' i
+    enforce i as = runSysOnly $ gasUnreduced i as $
+      ifExecutionFlagSet FlagDisablePact45
+      (mapM reduce as >>= enforce' i)
```

Comment:
Is it just GitHub, or do these arguments need indenting?

Reply:
Just github, they're definitely indented.